packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -63,6 +63,7 @@
     "@justeat/f-vue-icons": "3.x",
     "@justeat/f-wdio-utils": "1.x",
     "@vue/cli-plugin-babel": "5.0.0",
+    "core-js": "3.36.1",
```

Comment:
Not sure why, but had to add this as a devDep to prevent module resolution issues. I suspect something has changes in the dependency tree for some reason.